R/ftrlprox.default.r

```diff
@@ -49,7 +51,9 @@ ftrlprox.default <- function(x, y, lambda1, lambda2, alpha, beta=1, num_epochs=1
              y=as.double(ynum),
              m=as.integer(nrow(x)),
              n=as.integer(ncol(x)),
-             J=numeric(num_epochs),
+             z=double(ncol(x)),
```

Comment:
Initializing and sending in the z and nn vectors here will add them to the out object